icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2663,14 +2663,28 @@ class U_I18N_API LocalizedNumberFormatter
 #endif
 
 /**
- * The result of a number formatting operation. This class allows the result to be exported in several data types,
- * including a UnicodeString and a FieldPositionIterator.
+ * Represents all the grammatical class nouns that are supported by CLDR.
```

Comment:
is the enum in the right place and right form?

Reply:
Hmm, good question. I think you should make a new file called *unounclass.h* (a C header) and put it in there.

Reply:
done!